Makefile

```diff
@@ -107,6 +107,7 @@ update_dev_tools:
 	@echo "--> Downloading linters (this may take awhile)"
 	$(GOPATH)/src/github.com/alecthomas/gometalinter/scripts/install.sh -b $(GOBIN)
 	go get -u github.com/tendermint/lint/golint
+	go get -u github.com/walle/lll/...
```

Comment:
Please use the install script for this. the tendermint golint go get was only to override gometalinter's default.

Reply:
Done

Reply:
I can't use the install script, I need to drop it after that script run - I'm using `go get  github.com/alessio/lll/...`